### ISSUE TITLE
Add FAQ entry reg. „VC_PREFIX_INVALID“ error

### ISSUE THREAD

**Issue by Ein-Tim** [corona-warn-app/cwa-website#1343]
## Where to find the issue
n/a

## Describe the issue
Many people are trying to scan QR-Codes that they got with their vaccination shots. However, these are all no valid EU digital COVID certificates do the app doesn't accept these. 
The user thinks that the app is not working correctly.

See e.g. &

## Suggested change
Add a FAQ entry explaining this error and that the app is working correctly. Maybe also link to https://www.aponet.de/artikel/corona-digitaler-impfnachweis-ab-14-juni-in-apotheken-24262.amp so that the people know where to go to get their certificate.

**Comment by MikeMcC399** [comment]
@Ein-Tim 

I hope we will see an announcement of the 2.3.2 CWA version today on https://www.coronawarn.app/en/blog/ giving some details and context, since 2.3.2 was released to the stores yesterday. Perhaps there will be some explanation there about where to get certificates?

Has there been any official announcement of Twitter about the new version?

**Comment by Ein-Tim** [comment]
@MikeMcC399 

> Has there been any official announcement on Twitter about the new version?

No, only the [BSI](https://twitter.com/bsi_bund/status/1402339715991359488?s=21) tweeted that there is a new version.

**Comment by Ein-Tim** [comment]
@MikeMcC399 

FYI: https://twitter.com/rainerknirsch/status/1402530517002735618?s=21 says that they will publish a blog post tomorrow... 

@dsarkar Are you working on this?

**Comment by thomasaugsten** [comment]
This is correct release notes and blog post is planned for tomorrow.

**Comment by MikeMcC399** [comment]
https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/WarnApp/Warn_App.html describes the 2.3 release now.

It only says:
"Um ein Impfzertifikat in der App hinzuzufügen, scannen sie den QR-Code, den Sie bei der Impfung erhalten haben."

This is less information than described in https://www.coronawarn.app/en/faq/#vac_cert_how.

**Comment by Ein-Tim** [comment]
One more user (@Melancholyaeon ) [requesting](https://github.com/corona-warn-app/cwa-app-ios/issues/2871#issuecomment-857622291) a FAQ entry. He also brings up the valid point how people who don't have the yellow vaccination booklet (because they are only long time residents, etc.) will get their digital vaccination certificate.

**Comment by MikeMcC399** [comment]
@Ein-Tim 
I was just visiting my local pharmacy this afternoon, so I asked them whether they will be able to issue digital vaccination certificates. They said they don't have the software yet, but they had already read in the press that it should be possible from Monday onwards.

Regarding people without the yellow vaccination booklet: According to https://www.stmgp.bayern.de/coronavirus/impfung/ "Die Coronaschutzimpfung wird, wie jede andere Impfung auch, im Impfausweis oder in einer Impfbescheinigung dokumentiert." So if they don't have the yellow booklet they should receive some other document which they could use at the pharmacy.

**Comment by Ein-Tim** [comment]
@thomasaugsten said in https://github.com/corona-warn-app/cwa-app-ios/issues/2871#issuecomment-857654042 that "A detailed FAQ entry will follow soon".

**Comment by MikeMcC399** [comment]
@Ein-Tim 
Thanks for the message that a detailed FAQ will follow.  I guess we need to understand exactly when the detailed FAQ will be published as "soon" is a relative term.  If this is going to take longer, then perhaps in the meantime a short FAQ would be good?

Suggestion:
EN
What does the error VC_PREFIX_INVALID mean when I try to add a vaccination certificate?

The QR code you are scanning is not compatible with the newly-standardized certificate. Please wait for further announcements about where you will be able to get a compatible vaccination certificate.

DE
Was bedeutet der Fehler VC_PREFIX_INVALID, wenn ich versuche ein Impfzertifikat zuzufügen?

Der QR-Code, die Sie versuchen einzuscannen, ist mit dem erst vor kurzem festgelegtem Format nicht kompatibel. Bitte warten Sie auf weitere Ankündigungen, wie Sie in Zukunft ein kompatibles Zertifikat bekommen können.

**Comment by Ein-Tim** [comment]
@MikeMcC399 I'm up for anything I can quickly link from Twitter. Your suggestion looks good!
@dsarkar Could you give us an assumption when the FAQ entry will be published?

**Comment by melancholyaeon** [comment]
@Ein_Tim

Ty for listening.

There are a large number of non-EU people in Germany.... spouses,
high-skilled workers, students, ESL teachers, translators, journalists,
low-skilled people in gray market labor, expats, retirees, military,
researchers, refugees, all the illegal programmers at the start-ups in
Berlin, Munich & Hamburg....

I have no idea how or when we'll get our EU QR Impf Codes here in Hesse.
"Sometime in the future" I guess!

The finished grad students have to leave in September, but if they don't
have an EU Code or a yellow passport do they get stuck here & become
illegal aliens because they can't travel?

It's a political & economic issue; nothing you can solve.

Best wishes!

**Comment by Ein-Tim** [comment]
**Update:**

We will get a hotfix for this! See https://github.com/corona-warn-app/cwa-app-android/pull/3400.
The error message will then also include a link to a FAQ entry.

**Comment by MikeMcC399** [comment]
@Ein-Tim 
Thanks for highlighting the pending Android hotfix corona-warn-app/cwa-app-android#3400 which links to https://www.coronawarn.app/de/faq/#vac_cert_invalid.
I haven't seen any PR yet for the FAQ contents.

**Comment by dsarkar** [comment]
@Ein-Tim @MikeMcC399 @melancholyaeon FYI https://github.com/corona-warn-app/cwa-website/pull/1344

**Comment by MikeMcC399** [comment]
@melancholyaeon 

As you may have gathered if you read the rest of this thread, this is actually a technical forum about the Corona-Warn-App website, but anyway ...

> The finished grad students have to leave in September, but if they don't have an EU Code or a yellow passport do they get stuck here & become illegal aliens because they can't travel?

Most people who are travelling internationally are getting tested beforehand, so for instance to fly to USA from Germany it says you need a PCR test or an Antigen test which is not older than 3 days old. Vaccination is voluntary in Germany. 

The situation with vaccination certificates is still quite fluid and we expect that there will be some more announcements in the next days. So don't worry!
